pkg/bbgo/order_executor_general.go

```diff
@@ -170,6 +181,30 @@ func (e *GeneralOrderExecutor) CancelOrders(ctx context.Context, orders ...types
 	return err
 }
 
+func (e *GeneralOrderExecutor) FastSubmitOrders(ctx context.Context, submitOrders ...types.SubmitOrder) (types.OrderSlice, error) {
+	formattedOrders, err := e.session.FormatOrders(submitOrders)
+	if err != nil {
+		return nil, err
+	}
+	createdOrders, errIdx, err := BatchPlaceOrder(ctx, e.session.Exchange, formattedOrders...)
+	if len(errIdx) > 0 {
+		return nil, err
+	}
+	if IsBackTesting {
+		e.orderStore.Add(createdOrders...)
+		e.activeMakerOrders.Add(createdOrders...)
+		e.tradeCollector.Process()
+	} else {
+		go func() {
+			e.orderStore.Add(createdOrders...)
+			e.activeMakerOrders.Add(createdOrders...)
```

Comment:
might need to check if these are thread safe?

Reply:
yes. orderStore.Add is protected by mutex.
and activeMakerOrders.orders is a SyncOrderMap.